src/gazebo_mavlink_interface.cpp

```diff
@@ -957,6 +962,65 @@ void GazeboMavlinkInterface::IRLockCallback(IRLockPtr& irlock_message) {
   mavlink_interface_->send_mavlink_message(&msg);
 }
 
+void GazeboMavlinkInterface::ArucoMarkerCallback(ArucoMarkerPtr& arucoMarker_message) {
+  
+  mavlink_target_relative_t sensor_msg;
```

Comment:
These mavlink messages are not merged (https://github.com/mavlink/mavlink/pull/1943) Therefore, I don't think we can merge this PR into this repository

Reply:
The mavlink messages are now merged.